### ISSUE TITLE
Glitchy App on Android 11

### ISSUE THREAD

**Issue by meximan** [CollaboraOnline/online#3318]
I have an Android 11 Samsung Galaxy phone and the app is very glitchy with disappearing visuals at times and its not very responsive. It is very unstable. Help please. Other office software is trash and steals your personal info.

**Comment by meximan** [comment]
Just wanted to add, I tried other similar office apps from the Google play store and there is no visual or stability issues. It seems Collabora isn't completely optimized to run smoothly and quickly on android.

**Comment by rpear** [comment]
I am aware of issues too, looking at the software fixes that are being made it appears that a lot of effort is being made improving the mobile apps (or at least they are for iOS), I'm hoping to see a new release in the next few weeks, or month or so.

In the mean time, you could try the 2 versions of Collabora Office that come with F-Droid, I don't know if they are better or worse, let me know here if you try and have a better experience.   See https://www.collaboraoffice.com/releases-en/collabora-office-on-mobiles-supporting-password-protected-documents-and-available-on-f-droid/

**Comment by meximan** [comment]
Thank you for your response. I am using the F droid version. But I will try the snapshot version.

**Comment by meximan** [comment]
I am still having the same issues. Also suddenly my files don't save...

**Comment by yashpalgoyal1304** [comment]
yeah, android 11 seems verrrry alien to me in the way it is behaving

**Comment by timur-g** [comment]
This is old report but unfortunatelly never was properly confirmed, to know what was the trouble version.
Whatever, current stable Android app is 21.11 but there are new 23.05 snapshots released here: https://www.collaboraoffice.com/downloads/Collabora-Office-Android-Snapshot/.
(Not having snapshots in F-Droid is another known issue).
Please try again and write if you reproduce the issue and write exact steps or attach a screencast.

**Comment by rpear** [comment]
I no longer have an Android phone, so I can't help with this. I do use the Android app on a chromebook quite a bit and in general it is good.  However, the other day after I minimised and then maximised the app, the screen was just white, I have not been able to reproduce this.   If I find I can reproduce it, I would create a new ticket.  As far as I am concerned, I would close this ticket.
